app/models/service_binding.rb

```diff
@@ -9,29 +9,31 @@ class HostNotFound < RuntimeError
   end
 
   class << self
-    def create(instance_id, binding_id, app_id)
```

Comment:
out of curiosity, did you remove these just bc they're not being used? I wonder if it's worth leaving these if we might update to use the app GUID instead of the binding GUID at some point (I think that's why they are here now)

Reply:
This is only my opinion, but I think it would be more clear to remove them since they aren't used now, and add the `app_guid` parameter back once we're ready to actually use it in a future changeset.